.node-version

```diff
@@ -0,0 +1 @@
+18.14.0
```

Comment:
What's the reason for using a fully qualified version now instead of `18.x`?

Reply:
A `.node-version` file should generally be an exact version so that you're guarenteed consistent results in all build environments.

We should probably set up dependabot to bump this for us though. Also, I'll bump this to the latest version of 18.

Reply:
Yeah I was mostly worried about us not upgrading to the latest versions. Dependabot sounds good!